contracts/contracts/interfaces/IPreConfCommitmentStore.sol

```diff
@@ -40,17 +39,17 @@ interface IPreConfCommitmentStore {
         bytes sharedSecretKey;
     }
 
-    /// @dev Struct for all the information around encrypted preconfirmations commitment
-    struct EncrPreConfCommitment {
-        bool isUsed;
+    /// @dev Struct for all the information around unopened preconfirmations commitment
+    struct UnopenedCommitment {
+        bool isUsed; // Flag to check if the commitment is opened already
```

Comment:
Yea `isOpened` is better here imo since `OpenedCommitment.IsUsed` has a different meaning